src/components/BrowserSnifferService.js

```diff
@@ -103,6 +109,7 @@
         webkit: webkit,
         mac: mac,
         safari: safari,
+        chrome: chrome,
```

Comment:
Put a comment  like for the ios property : 
 `// false or chrome version number